riak/transports/tcp/connection.py

```diff
@@ -228,10 +222,11 @@ def close(self):
                 # shutdown() method due to the SSL lib
                 try:
                     self._socket.shutdown(socket.SHUT_RDWR)
-                except IOError as e:
-                    # NB: sometimes this is the exception if the initial
-                    # connection didn't succeed correctly
-                    if e.errno != errno.EBADF:
-                        raise
+                except EnvironmentError:
```

Comment:
How is `EnvironmentError` related to `IOError`? Will we catch the same exceptions?